src/ChatList/props.tsx

```diff
@@ -0,0 +1,49 @@
+import { HTMLAttributes } from 'react';
+
+import { Properties } from 'csstype';
+
+export interface MessageProps {
```

Comment:
any reason we don't want to keep these props in their component folders?

Reply:
Good question! Thats why I need this reviewed!

I'm making a `util/mocks.ts` file for sharing "server served data" across all these components.

Write them out very will in *one place* to share across storybook